### ISSUE TITLE
Job applying is not working 

### ISSUE THREAD

**Issue by waqaramjad** [aminblm/linkedin-application-bot#30]
There are many changes that happpens in linked structure recently due to which tag selection is not working, Can you please look into this and update the script?

**Comment by tehcmanmax** [comment]
@waqaramjad true. It looks that the project has been abandoned. Alternative thing is took look at the html yourself and fix the script
